packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1720,8 +1720,25 @@ export class ApexDebug extends LoggingDebugSession {
 
       // log to console and notify client
       this.logEvent(message);
+      let reason = '';
+
+      // if breakpointid is found in exception breakpoint cache
+      // set the reason for stopped event to that reason
+      if (message.sobject.BreakpointId) {
+        const cache: Map<
+          string,
+          string
+        > = this.myBreakpointService.getExceptionBreakpointCache();
+        cache.forEach((value, key) => {
+          if (value === message.sobject.BreakpointId) {
+            if (key.indexOf('Exception') !== -1) {
```

Comment:
Did you run into a scenario where the typeref didn't have the substring `Exception`?

Reply:
No, think I forgot that the cache only holds the exception breakpoints, will remove the check.